cli/types/cypress.d.ts

```diff
@@ -3074,7 +3074,16 @@ declare namespace Cypress {
     xhrUrl: string
   }
 
-  interface TestConfigOverrides extends Partial<Pick<ConfigOptions, 'animationDistanceThreshold' | 'blockHosts' | 'defaultCommandTimeout' | 'env' | 'execTimeout' | 'includeShadowDom' | 'numTestsKeptInMemory' | 'pageLoadTimeout' | 'redirectionLimit' | 'requestTimeout' | 'responseTimeout' | 'retries' | 'screenshotOnRunFailure' | 'slowTestThreshold' | 'scrollBehavior' | 'taskTimeout' | 'viewportHeight' | 'viewportWidth' | 'waitForAnimations' | 'experimentalSessionAndOrigin'>>, Partial<Pick<ResolvedConfigOptions, 'baseUrl'>> {
+  interface SuiteConfigOverrides extends Partial<
+    Pick<ConfigOptions, 'animationDistanceThreshold' | 'blockHosts' | 'defaultCommandTimeout' | 'env' | 'execTimeout' | 'includeShadowDom' | 'numTestsKeptInMemory' | 'pageLoadTimeout' | 'redirectionLimit' | 'requestTimeout' | 'responseTimeout' | 'retries' | 'screenshotOnRunFailure' | 'slowTestThreshold' | 'scrollBehavior' | 'taskTimeout' | 'viewportHeight' | 'viewportWidth' | 'waitForAnimations' | 'experimentalSessionAndOrigin'>
+  >, Partial<Pick<ResolvedConfigOptions, 'baseUrl' | 'testIsolation'>> {
+    browser?: IsBrowserMatcher | IsBrowserMatcher[]
+    keystrokeDelay?: number
+  }
+
+  interface TestConfigOverrides extends Partial<
```

Comment:
Can SuiteConfigOverrides be something like `TestConfigOverrides & { testIsolation?: bool }`, rather than relisting everything that's shared between the two? My eyes crossed trying to find the difference between these two.

Reply:
I have left it like this because I had a similar suggestion and the Typescript 'hover' message wasn't super clear
|  (Similar to) your suggestion | As Is |
| -- | --|
<img width="1184" alt="Screen Shot 2022-11-08 at 11 54 36 AM" src="https://user-images.githubusercontent.com/14099737/200678326-9b934025-60c5-4dd7-b609-2feed710cc63.png">|<img width="647" alt="Screen Shot 2022-11-08 at 3 21 38 PM" src="https://user-images.githubusercontent.com/14099737/200678346-4faefe2d-8951-4046-9aec-5678d2eca275.png">

Thoughts? I don't have a strong opinion.

Reply:
I'll be honest: I can't make heads or tails of *either* of those errors. Typestcript errors are always unreadable. :woman_shrugging: 

I'm fine leaving it as is if it looks clearer to you.